spec/lib/krikri/aggregation_entity_behavior_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'spec_helper'
+
+describe Krikri::AggregationEntityBehavior do
+
+  before do
+    # The harvest activity is the activity that generated the entities upon
+    # which the mapping activity below has acted.  I suppose we could mock the
+    # call that will happen to EntityConsumer#assign_generator_activity! when
+    # Krikri::Mapper::Agent is instantiated, but that seems like jumping
+    # through more hoops and making this less comprehensible than it is now.
+    DatabaseCleaner.clean_with(:truncation)
+    create(:krikri_harvest_activity)
+  end
+
+  describe '#entities' do
+
+    let(:mapping_activity_uri) do
+      (RDF::URI(Krikri::Settings['marmotta']['ldp']) /
```

Comment:
I can't really explain this, except to say that I found working with RSpec and mocking up all of the necessary function calls to be very confusing, at least using what was in place before this feature branch was started.  (And I started out simply trying to emulate what was there originally.)  I found that I had to dig quite a lot into the internals of methods I wasn't testing, and, instead of abstracting the process and making it easier to read, a lot of the mocks made it more difficult and time-consuming.  I probably copied and pasted something from somewhere else and was trying to shoehorn it in to fit within the context of this test.  I'll have another look at this as soon as I can today.